data/ecosystem/vendors.yaml

```diff
@@ -237,6 +237,13 @@
   contact: 'hello@openobserve.ai'
   oss: true
   commercial: true
+- name: OpenText
+  distribution: true
```

Comment:
I think this is not a distribution, as you're sending data through any OTel Collector, not your own distribution of it.

Reply:
@theletterf at the moment we have our own distribution of OpenTelemetry Collector. Hence set distribution as true.